src/components/hmi_message_handler/test/hmi_message_handler_impl_test.cc

```diff
@@ -189,6 +189,38 @@ TEST_F(HMIMessageHandlerImplTest, SendMessageToHMI_Success) {
   EXPECT_TRUE(waiter.WaitFor(1, 100));
 }
 
+TEST(WebsocketSessionTest, SendMessage_Success) {
```

Comment:
@SNiukalov Rename it please according to the best practices

Reply:
@AByzhynar 
Thanks, fixed in: https://github.com/smartdevicelink/sdl_core/pull/3191/commits/ccc2795f8cdbaf2f543d92b38f2183831538b3c9